### ISSUE TITLE
Build failure on github actions

### ISSUE THREAD

**Issue by fire** [maiself/godot-python-extension#28]
```
Auto-detected 3 CPU cores available for build parallelism. Using 3 cores by default. You can override it with the -j argument.
Building for architecture universal on platform macos
KeyError: ('macos', 'universal'):
  File "/Users/runner/work/godot-python-extension/godot-python-extension/SConstruct", line 289:
    prepared_python_config = prepare_python.platform_configs[(env['platform'], env['arch'])]
```

**Comment by maiself** [comment]
See #26

Tho windows builds are now failing for an unknown reason.

**Comment by fire** [comment]
The error message says something about casting error in c++ `(`

**Comment by maiself** [comment]
Except the code hasn't changed and builds fine on all platforms otherwise. I'm guessing the compiler changed, but I'm not sure how to look into it.

**Comment by Ivorforce** [comment]
The windows compile bug is unrelated; it stopped working with a windows-latest update and was fixed in #29.
It also disables the macOS runner for builds to pass, which will be re-enabled with the merge of #26.

**Comment by Ivorforce** [comment]
This can be closed now.
